firmware/joel/app.cpp

```diff
@@ -0,0 +1,11 @@
+// app.cpp
+#include <iostream>

```

Comment:
To get started, you want this to be as simple as possible. Just like `int main() { return 2; }` otherwise you have to solve loading/address fixup which is its own hard problem.